tools/build/cross-build/include/common/sys/_types.h

```diff
@@ -45,3 +45,6 @@
  * Neither GLibc nor macOS define __va_list but many FreeBSD headers require it.
  */
 typedef __builtin_va_list __va_list;
+
+/* Needed for opensolaris compat. */
+typedef	__int64_t	off64_t;
```

Comment:
this is fine now, but may cause some grief in the future when I land our better compat with open solaris off64_t stuff